lib/vanagon/platform/windows.rb

```diff
@@ -8,7 +8,13 @@ class Windows < Vanagon::Platform
       def generate_package(project)
         target_dir = project.repo ? output_dir(project.repo) : output_dir
         ["mkdir -p output/#{target_dir}",
-        "touch output/#{target_dir}/#{project.name}-#{project.version}-#{project.release}-#{@architecture}.msi"]
+        "mkdir -p $(tempdir)/#{project.name}/tools",
+        "cp #{project.name}.nuspec $(tempdir)/#{project.name}/",
+        "cp chocolateyInstall.ps1 chocolateyUninstall.ps1 $(tempdir)/#{project.name}/tools/",
+        "cp file-list $(tempdir)/#{project.name}/tools/file-list.txt",
+        "gunzip -c #{project.name}-#{project.version}.tar.gz | '#{@tar}' -C '$(tempdir)/#{project.name}/tools' --strip-components 1 -xf -",
+        "(cd $(tempdir)/#{project.name} ; #{self.drive_root}/ProgramData/chocolatey/bin/choco.exe pack #{project.name}.nuspec)",
+        "cp $(tempdir)/#{project.name}/*.nupkg ./output/#{target_dir}"]
```

Comment:
As mentioned earlier, you could eliminate these two steps with a well formed pack command that puts the package in target directory on build.